lib/periodicChecks.js

```diff
@@ -174,7 +175,38 @@ const ensureAllIssuesHaveProjects = async (context) => {
   });
 };
 
+/**
+ * 1. Check if the new issue have a project assigned.
+ * 2. If not then it will ping Core Maintainers
+ * @param {import('probot').Context} context
+ */
+const ensureNewIssuesHaveProjects = async(context) =>{
+  await sleep(20000);
+  const issue = context.payload.issue;
+  const allProjectCards = await getAllProjectCards(context);
+
+  const doesIssueHaveProject = allProjectCards.some((card) => {
+    return card.content_url.endsWith(issue.number);
+  });
```

Comment:
Isn't there a different way to do this? Can't we get the project to which the issues is assigned directly from the issue object?

Reply:
No Sir, There is no such way As issue object does not have any field related to project.